command/acl_bootstrap.go

```diff
@@ -82,8 +88,12 @@ func (c *ACLBootstrapCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Take the provided ACL token and prepare it for the query.
+	btkn := api.BootstrapRequest{}
+	btkn.Secret = bootstraptoken
```

Comment:
Style and consistency nit: this is a small struct so it's very readable to instantiate and populate its fields all at once:

```suggestion
	btkn := &api.BootstrapRequest{Secret: bootstraptoken}
```